diff --git a/lib/commonjs/native.js b/lib/commonjs/native.js
index 89083ca728038bb4fb9078dfb5cb2aa75cdef162..9afb7056512e88afa6eb963b15724a62f958e3c1 100644
--- a/lib/commonjs/native.js
+++ b/lib/commonjs/native.js
@@ -443,7 +443,7 @@ const isChannelSwitched = () => {
 exports.isChannelSwitched = isChannelSwitched;
 const getFingerprintHash = () => {
   const constants = _NativeHotUpdater.default.getConstants();
-  return constants.FINGERPRINT_HASH;
+  return constants.FINGERPRINT_HASH ?? null;
 };
 exports.getFingerprintHash = getFingerprintHash;
 const getInstallId = () => {
diff --git a/lib/module/native.js b/lib/module/native.js
index b9f1a8ffee952e19085ae825a6ab60fa727545bc..a9fa4d2b71c24dd3c536fe507d201b1d66c267b2 100644
--- a/lib/module/native.js
+++ b/lib/module/native.js
@@ -408,7 +408,7 @@ export const isChannelSwitched = () => {
  */
 export const getFingerprintHash = () => {
   const constants = HotUpdaterNative.getConstants();
-  return constants.FINGERPRINT_HASH;
+  return constants.FINGERPRINT_HASH ?? null;
 };
 export const getInstallId = () => {
   const nativeModule = HotUpdaterNative;
diff --git a/src/native.ts b/src/native.ts
index ef0d8b15c5b5e9e183bd9311e3a092ad6c3ee26b..882073d77c964b77c254ae777d450dcc6c12b6e2 100644
--- a/src/native.ts
+++ b/src/native.ts
@@ -687,5 +687,5 @@ export const isChannelSwitched = (): boolean => {
  */
 export const getFingerprintHash = (): string | null => {
   const constants = HotUpdaterNative.getConstants();
-  return constants.FINGERPRINT_HASH;
+  return constants.FINGERPRINT_HASH ?? null;
 };
